server/app.js

```diff
@@ -201,57 +203,13 @@ module.exports = function createApp ({ signInService }) {
     })
   })
 
-  // JWT token refresh
-  app.use(async (req, res, next) => {
-    let axiosHeaders = {}
-
-    if (req.user && req.originalUrl !== '/logout') {
-      if (req.user.refreshTime && new Date() > req.user.refreshTime) {
-        try {
-          const newToken = await signInService.getRefreshedToken(req.user)
-          req.user.token = newToken.token
-          req.user.refreshToken = newToken.refreshToken
-          if (!config.settings.reduceStdoutNoise) {
-            log.info(`existing refreshTime in the past by ${new Date().getTime() - req.user.refreshTime}`)
-            log.info(
-              `updating time by ${newToken.refreshTime - req.user.refreshTime} from ${req.user.refreshTime} to ${
-                newToken.refreshTime
-              }`
-            )
-          }
-          req.user.refreshTime = newToken.refreshTime
-        } catch (error) {
-          log.error(`Token refresh error: ${req.user.username}`, error.stack)
-          return res.redirect('/logout')
-        }
-      }
-
-      axiosHeaders = {
-        Authorization: `Bearer ${req.user.token}`
-      }
-    }
-    service.reset()
-    service.register({
-      onRequest (config) {
-        config.headers = {
-          ...config.headers,
-          ...axiosHeaders
-        }
-        return config
-      }
-    })
-    return next()
-  })
-
```

Comment:
So this never worked? Is this where we used the middleware library? Can that go?

Reply:
No, two separate things. The passport express lib doesn't handle refresh functionality at all.